.gitignore

```diff
@@ -118,3 +118,10 @@ venv.bak/
 # Misc backups
 *.bak
 
+# Excel lockfiles
+legal-test-fixture/test/spreadsheets/.~lock*
+
+# Copied files for test fixture
+legal-test-fixture/legal_api/exceptions/*
```

Comment:
import as library and remove these

Reply:
Resolving on new PR (rebasing is fubared)